Detectors/TRD/calibration/src/DCSProcessor.cxx

```diff
@@ -0,0 +1,246 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "TRDCalibration/DCSProcessor.h"
+#include "DetectorsCalibration/Utils.h"
+
+using namespace o2::trd;
+using namespace o2::dcs;
+
+using DeliveryType = o2::dcs::DeliveryType;
+using DPID = o2::dcs::DataPointIdentifier;
+using DPVAL = o2::dcs::DataPointValue;
+
+void TRDDCSMinMaxMeanInfo::print() const
```

Comment:
Don't use people ostream operators these days? ;-) Maybe it's not quite in line with O2 coding standards.

On a serious note: I guess this should also log the number of seen data points.